### ISSUE TITLE
switch from \\n to \n in regex separators

### ISSUE THREAD

**Issue by robstoll** [loewenfels/dep-graph-releaser#65]
It is a bit tricky to see for a user if she has inadvertently put newlines into a regex argument. 
Since a newline never makes sense in a regex which matches groupId:artifactId, we could automatically cleanup (similar to #53).

Currently we use `\\n` as escaped newline but that is not really necessary. We can use `\n` as escaped newline in JSON, we only have to make sure we don't use an actual newline in the json data.

**Comment by robstoll** [comment]
Also, a newline at the beginning and at the end of an argument should be ignored. This way one can define jobMapping as follows for instance:
```
"-DjobMapping=" +/
     com....=xy
     com...=ab
     /.stipIndent()
```

**Comment by robstoll** [comment]
resolved with 92c15ed
